bundles/net.tourbook/src/net/tourbook/preferences/PrefPageImport.java

```diff
@@ -42,6 +44,7 @@
 
    public static final String    ID          = "net.tourbook.preferences.PrefPageImport"; //$NON-NLS-1$
 
+   private final IPreferenceStore _prefStore  = TourbookPlugin.getPrefStore();

```

Comment:
This code is not autoformatted, here it's just one space but it looks like you have not activate the autosave feature in Eclipse http://mytourbook.sourceforge.net/mytourbook/index.php/development/dev-ide/code-style which will autoformat the code when it is saved.

This field is also not used but not used fields are not automatically removed. It could be set to remove unused fields (as I remember) but this would be too dangerous especially when you have not finished the code.
![mt-not-used](https://user-images.githubusercontent.com/1283445/92484600-2f835980-f1ea-11ea-8e0e-ecfca76edcf0.png)